content/en/docs/tasks/configure-pod-container/image-volumes.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Use an Image Volume With a Pod
+reviewers:
+content_type: task
+weight: 210
+min-kubernetes-server-version: v1.31
+---
+
+<!-- overview -->
+
+{{< feature-state feature_gate_name="ImageVolume" >}}
+
+This page shows how to configure a pod using image volumes. This allows you to
+mount content from OCI registries inside containers.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% thirdparty-content single="true" %}}
```

Comment:
Just noticed something: there's no third party reference here. Did we plan to include a list of known compatible runtimes? If not, omit this line.

Reply:
For alpha, happy to ship this even with the snag I spotted here.

Reply:
Ah okay, I remove the line for now.